components/dashboard/src/WhatsNew.tsx

```diff
@@ -0,0 +1,75 @@
+}
+
+export function WhatsNew(props: { visible: boolean, onClose: () => void }) {
+    const {user, setUser} = useContext(UserContext);
+    const internalClose = async (switchToCode?: boolean) => {
+        if (!user) {
+            return;
+        }
+        const additionalData = user.additionalData || {};
+        additionalData.whatsNewSeen = {
+            ...additionalData.whatsNewSeen,
+            [news]: new Date().toISOString()
+        }
+        if (switchToCode) {
+            const ideSettings = additionalData.ideSettings || {};
+            ideSettings.defaultIde = 'code';
+        }
+        await getGitpodService().server.updateLoggedInUser({
+            additionalData
+        });
+        setUser(user);
+        props.onClose();
+    } 
+    return <Modal visible={props.visible} onClose={internalClose}>
+        <h3 className="pb-4">What's New 🎁</h3>
+        <div className="border-t border-gray-200 -mx-6 px-6 py-4">
+            <p className="pb-2 text-gray-900 text-base">New Dashboard</p>
+            <p className="pb-2 text-gray-500 text-sm">We have made some layout changes on the dashboard to improve the overall user experience of the product.</p>
+        </div>
+        <div className="border-t border-b border-gray-200 -mx-6 px-6 py-4">
+            <p className="pb-2 text-gray-900 text-base">VS Code</p>
+            <p className="pb-4 text-gray-500 text-sm">We are changing the default IDE to VS Code.</p>
+            <ol className="pb-2 text-gray-500 text-sm list-outside list-decimal space-y-2">
+                <li className="ml-5">
+                    <div>
+                        <p className="text-gray-500 text-sm">We're preserving all <span className="font-bold">user settings and extensions</span>.</p>
+                        <p className="text-gray-400 text-xs">Extensions you have manually uploaded are not transfered. You'll need to search and install those extensions through the extension panel in VS Code.</p>
+                    </div>
+                </li>
+                <li className="ml-5">
+                    <div>
+                        <p className="text-gray-500 text-sm">We've reduced the number of <span className="font-bold">pre-installed extensions</span>.</p>
+                        <p className="text-gray-400 text-xs">The old editor would come with extensions for the most popular programming languages pre-installed. You are now freed from that additional bloat but should consider installing any extensions you miss.</p>
+                    </div>
+                </li>
+                <li className="ml-5">
+                    <div>
+                        <p className="text-gray-500 text-sm">You can still <span className="font-bold">switch the editor back</span> to the old one.</p>
+                        <p className="text-gray-400 text-xs">In case you run into trouble with the new IDE, you can go to the settings and switch back to the old editor (Theia).</p>
```

Comment:
```suggestion
                        <p className="text-gray-400 text-sm">In case you run into trouble with the new IDE, you can go to the settings and switch back to the old editor (Theia).</p>
```